packages/core-utils/src/query-params.js

```diff
@@ -267,14 +267,14 @@ const queryParams = [
     routingTypes: ["ITINERARY"],
     default: "QUICK",
     selector: "DROPDOWN",
-    label: "Optimize for",
+    label: "I prefer",
```

Comment:
This language is kind of niche so I feel like the text for these labels should come from the config file.  Also our marketing team likes to change this kind of thing often so would be helpful in that sense too.  Don't see this as blocking to merge this code though

Reply:
Customization of these strings is already covered by https://github.com/opentripplanner/otp-ui/pull/273.